mypy/newsemanal/semanal.py

```diff
@@ -3984,6 +3892,12 @@ def lookup_fully_qualified_or_none(self, fullname: str) -> Optional[SymbolTableN
             self.record_incomplete_ref()
         return result
 
+    def builtin_type(self, fully_qualified_name: str) -> Instance:
+        sym = self.lookup_fully_qualified(fully_qualified_name)
+        node = sym.node
+        assert isinstance(node, TypeInfo)
+        return Instance(node, [AnyType(TypeOfAny.special_form)] * len(node.defn.type_vars))
+
     def object_type(self) -> Instance:
         return self.named_type('__builtins__.object')
 
```

Comment:
I think `class_type()` (can't add comment there) rather belongs to `Other helpers`.